src/pages/Login.js

```diff
@@ -0,0 +1,37 @@
+import "../styles/auth.css";
+import {NavMenu} from "../components/NavMenu";
+import {Link} from "react-router-dom";
+const Login = () => {
+    return (
+        <>
+        <NavMenu />
+         <section className="form-ctn br-md">
+        <form>
+            <h2 className="text-center mg-bottom-md">Login</h2>
+            <div className="form-control">
+                <label htmlFor="email" className="fw-bold">Email address</label>
+                <input type="text" name="email" placeholder="name@gmail.com" />
+            </div>
+            <div className="form-control">
+                <label htmlFor="password" className="fw-bold">Password</label>
+                <input type="password" name="password" placeholder="&lowast;&lowast;&lowast;&lowast;&lowast;&lowast;&lowast;&lowast;" />
```

Comment:
- here you forgot to add ```id``` attribute.

```suggestion
                <input type="password" name="password" id="password" placeholder="&lowast;&lowast;&lowast;&lowast;&lowast;&lowast;&lowast;&lowast;" />
```